### PR TITLE
Make raw sausage cookable

### LINKED ISSUE #27240: No way to cook raw sausage
I have made raw sausage in game but to my dismay there are no recipes to cook it. (and yes I have a fully loaded charcoal smoker) Am I just missing something here? If I am missing something I think there should be more clarification on how to cook it.

### PR DESCRIPTION
**Pull request by Pupsi-Mupsi**
#### Summary

`SUMMARY: Bugfixes "Change raw sausage description to be more precise"`

#### Purpose of change

Fixes #27240

### CODE REVIEW

File: data/json/items/comestibles.json [hunk 1]
```diff
@@ -2389,7 +2389,7 @@
     "calories": 278,
     "healthy": -1,
     "parasites": 32,
-    "description": "A hefty raw sausage, prepared for smoking.",
+    "description": "A hefty raw sausage, prepared for smoking. Needs to be loaded into a smoking rack to cook it.",
```
Comment (ZhilkinSerg) [inline data/json/items/comestibles.json]:
```suggestion
     "description": "A hefty raw sausage, prepared for smoking.  Needs to be loaded into a smoking rack to cook it.",
```

**Comment by Pupsi-Mupsi** [comment]
Sure - good idea. Would you rather replace the existing raw sausage or just create a bunch of new ones.

**Comment by Rivet-the-Zombie** [comment]
Keep what we already have and add a recipe and new item for 'cooked sausage' using standard cooking gear.

**Comment by Rivet-the-Zombie** [comment]
I'd meant a recipe that uses standard cooking equipment/heat source _with the existing 'raw sausage' item as its sole ingredient_. Basically just add a single recipe and a single 'cooked sausage' item so that the current 'raw sausage' can be cooked in its extant form.

**Comment by Pupsi-Mupsi** [comment]
@Rivet-the-Zombie 
> 
> I'd meant a recipe that uses standard cooking equipment/heat source _with the existing 'raw sausage' item as its sole ingredient_. Basically just add a single recipe (https://github.com/CleverRaven/Cataclysm-DDA/pull/27243/commits/ad70ee315953993acdb6fa48c28dd759b747efe4) and a single 'cooked sausage' item (https://github.com/CleverRaven/Cataclysm-DDA/pull/27243/commits/10210b0f5ae3e6b5a1b170be1f123d604f03dbab) so that the current 'raw sausage' can be cooked in its extant form.

**Comment by Pupsi-Mupsi** [comment]
I open a new RP one to fix the mess i made!

**Pull request closed without merging**